code/modules/projectiles/special.dm

```diff
@@ -1400,3 +1400,42 @@ ABSTRACT_TYPE(/datum/projectile/special)
 
 	on_pointblank(var/obj/projectile/O, var/mob/target)
 		src.on_hit(target, O = O)
+
+/datum/projectile/special/psi_bolt
+	name = "psionic-kinetic paralysis bolt"
+	sname = "psi bolt"
+	icon_state = "psi_bolt"
+	damage = 0
+	dissipation_delay = 7
+	hit_ground_chance = 50
+	shot_sound = 'sound/weapons/psi_bolt.ogg'
+
+	on_launch(obj/projectile/O)
+		var/mob/living/critter/mindeater/mindeater = O.shooter || O.mob_shooter
+		mindeater.reveal()
+
+	on_pre_hit(atom/hit, angle, obj/projectile/O)
+		. = ..()
+		if (istype(hit, /mob/living/critter/mindeater))
+			return TRUE
+
+	on_hit(atom/hit, angle, obj/projectile/O)
+		. = ..()
+		var/mob/living/L = hit
+		if (!istype(L))
+			return
+		if (istype(L, /mob/living/critter/mindeater))
+			return
+		L.changeStatus("staggered", 0.75 SECONDS)
+		L.setStatus("psi_bolt_slow", 5 SECONDS)
```

Comment:
This slow is almost unnoticeable, especially with sprint.

Reply:
Was bugged, LMK if you think it needs to be increased